handler/src/main/java/io/netty/handler/internal/svm/ssl/JdkDefaultApplicationProtocolNegotiatorSubstitution.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.internal.svm.ssl;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(className = "io.netty.handler.ssl.JdkDefaultApplicationProtocolNegotiator")
+final class JdkDefaultApplicationProtocolNegotiatorSubstitution {
+
+    private JdkDefaultApplicationProtocolNegotiatorSubstitution() {
+    }
+
+    @Alias
+    public static JdkDefaultApplicationProtocolNegotiatorSubstitution INSTANCE;
```

Comment:
why is this public ?

Reply:
It fits with the target class. It's only purpose is to make the instance variable accessible in `io.netty.handler.internal.svm.ssl.JdkSslContextSubstitution`

Reply:
I see... I may not know enough about graalvm substitutions to give good feedback but shouldn't it also be "final" to match the original one ?